Sources/Valet/Internal/Keychain.swift

```diff
@@ -48,216 +48,173 @@ internal final class Keychain {
             var secItemQuery = attributes
             secItemQuery[kSecAttrAccount as String] = canaryKey
             secItemQuery[kSecValueData as String] = Data(canaryValue.utf8)
-            _ = SecItem.add(attributes: secItemQuery)
+            try? SecItem.add(attributes: secItemQuery)
```

Comment:
the side effect is invisible to the customer. I have a test to prove that the sentinel I'm writing doesn't appear in `allKeys`.

You're right that this is weird, but I think it's okay because the customer never has to know it happened.